linera-execution/src/wasm/wasmer.rs

```diff
@@ -48,6 +51,22 @@ use wasmer::{
 use wasmer_middlewares::metering::{self, Metering, MeteringPoints};
 use wit_bindgen_host_wasmer_rust::Le;
 
+/// An [`Engine`] instance configured to run application services.
+static SERVICE_ENGINE: Lazy<Engine> = Lazy::new(|| {
+    let compiler_config = Singlepass::default();
+    EngineBuilder::new(compiler_config).into()
+});
+
+/// A cache of compiled contract modules, with their respective [`Engine`] instances.
+///
+/// Each [`Module`] needs to have a separate [`Engine`] instance, otherwise Wasmer
```

Comment:
I think this comment is now misplaced?

Reply:
Yep. I moved it to `CachedContractModule` instead, and tweaked it a bit.